type-definitions/almin.js.flow

```diff
@@ -25,14 +24,19 @@ declare class Context {
     getState(): mixed;
     onChange(onChangeHandler: (changingStores: Array<Store>) => mixed): Function;
     useCase(useCase: UseCase): UseCaseExecutor;
-    onWillExecuteEachUseCase(handler: (useCase: UseCase, args: mixed) => mixed): void;
-    onDispatch(handler: (payload: DispatcherPayload) => mixed): Function;
-    onDidExecuteEachUseCase(handler: (useCase: UseCase) => mixed): void;
-    onCompleteExecuteEachUseCase(handler: (useCase: UseCase) => mixed): void;
-    onErrorDispatch(errorHandler: (payload: UseCaseErrorPayload) => mixed): (this: Dispatcher) => mixed;
+    onWillExecuteEachUseCase(handler: (payload: DispatcherPayload, meta: DispatcherPayloadMeta) => mixed): Function;
+    onDispatch(handler: (payload: DispatcherPayload, meta: DispatcherPayloadMeta) => mixed): Function;
+    onDidExecuteEachUseCase(handler: (payload: DispatcherPayload, meta: DispatcherPayloadMeta) => mixed): Function;
+    onCompleteExecuteEachUseCase(handler: (payload: DispatcherPayload, meta: DispatcherPayloadMeta) => mixed): Function;
+    onErrorDispatch(handler: (payload: DispatcherPayload, meta: DispatcherPayloadMeta) => mixed): Function;
```

Comment:
Apply unified arguments to `Context#on*` methods.

We will improve this maybe by preparing each Payload class

- `WillExecutePayload` class
- `DidExecutePayload` class
- `CompleteExecutePayload` class